mmv1/products/compute/api.yaml

```diff
@@ -2852,6 +2852,18 @@ objects:
         description: |
           The URL of the network to which this backend service belongs.
           This field can only be specified when the load balancing scheme is set to INTERNAL.
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
Is this field intended to be `beta` only? I don't understand how the encoder/decoder would not be needed in the GA case

Reply:
Yes, it is supposed to be `beta` only. I added a `min_version` field now, so should be fixed. Please take another look.